### PR TITLE
Update 3.7.0b5 to 3.7.0

### PR DESCRIPTION
**Pull request by tao12345666333**
Update 3.7.0b5 to [2018-06-27 released 3.7.0](https://www.python.org/ftp/python/3.7.0/).

### CODE REVIEW

**Comment by jedie** [comment]
When will the Python 3.7 image be available?!?

**Comment by ian-axelrod** [comment]
pls 2 merge

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -2,8 +2,9 @@
 set -Eeuo pipefail
 
 declare -A aliases=(
-	[3.7-rc]='rc'
-	[3.6]='3 latest'
+	[3.8-rc]='rc'
+	[3.7]='3 latest'
+	[3.6]='3.6'
```
Comment (RazerM) [inline generate-stackbrew-library.sh]: Isn't this line redundant?

Comment (tao12345666333) [reply generate-stackbrew-library.sh]: @RazerM  Thanks. I have removed it.

**Comment by chingc** [comment]
Is this waiting on something?

**Comment by yosifkit** [comment]
Apologies for the delay, it has been a busy week.

**Comment by yosifkit** [comment]
For those wondering why the merge has not produced new images on the Docker Hub:

> ## See a change merged here that doesn't show up on Docker Hub yet?
>
>Check [the "library/python" manifest file in the docker-library/official-images repo](https://github.com/docker-library/official-images/blob/master/library/python), especially [PRs with the "library/python" label on that repo](https://github.com/docker-library/official-images/labels/library%2Fpython).
>
>For more information about the official images process, see the [docker-library/official-images readme](https://github.com/docker-library/official-images/blob/master/README.md).
>
> \- [readme](https://github.com/docker-library/python/tree/341c752e5435f4cf4c008fbae67ae4b5b6209a02#see-a-change-merged-here-that-doesnt-show-up-on-docker-hub-yet)

**Comment by Gr1N** [comment]
@yosifkit so anyone need to create issue or PR in `docker-library/official-images` repo? Or Docker organization will handle this?

**Comment by yosifkit** [comment]
I'll be doing one shortly. :smile:

**Comment by tao12345666333** [comment]
@yosifkit Thanks.

**Pull request merged**